Fix API spec and add missing enpoints for user

EstebanOlmedo opened the pull request:
Correct the definition of some schemas that were wrong inside the API specification file. Add '/user/login' and '/user' endpoints to the api.yml file.

DanielMontesGuerrero commented on api.yml:
```diff
@@ -259,7 +281,7 @@ paths:
     parameters:
       - $ref: '#/components/parameters/operation'
-    get:
+    patch:
```
shouldn't this be post?

DanielMontesGuerrero commented on api/src/api/routes/operation.ts:
```diff
@@ -16,27 +16,27 @@ operationRouter.get('/analyze/:operation', (async(req, res, next) => {
                 cmd: '',
-                id1: '',
-                id2: '',
+                blobName1: '',
```
can we name it blobNameAfter/Prev, like the python code

EstebanOlmedo committed:
Change names according to backend

EstebanOlmedo committed:
Fix API spec and add missing enpoints for user

Correct the definition of some schemas that were wrong inside the API
specification file. Add '/user/login' and '/user' endpoints to the
api.yml file.
